code/modules/awaymissions/mission_code/Academy.dm

```diff
@@ -67,7 +67,7 @@
 	if(!current_wizard)
 		return
 	spawn(0)

```

Comment:
The guideline I follow is

Does everything calling this proc not want to wait for it's sleeps?

Y: set waitfor = FALSE
N: INVOKE_ASYNC